libraries/Microsoft.Bot.Builder.LanguageGeneration/Expander.cs

```diff
@@ -263,16 +268,36 @@ public override List<string> VisitNormalTemplateString([NotNull] LGFileParser.No
                     case LGFileParser.DASH:
                         break;
                     case LGFileParser.ESCAPE_CHARACTER:
-                        result = StringListConcat(result, new List<string>() { Regex.Unescape(node.GetText()) });
+                        result = StringListConcat(result, EvalEscape(node.GetText()));
                         break;
                     case LGFileParser.EXPRESSION:
                         result = StringListConcat(result, EvalExpression(node.GetText()));
                         break;
-                    case LGFileParser.TEMPLATE_REF:
-                        result = StringListConcat(result, EvalTemplateRef(node.GetText()));
+                    default:
+                        result = StringListConcat(result, new List<string>() { node.GetText() });
+                        break;
+                }
+            }
+
+            return result;
+        }
+
+        public override List<string> VisitMultilineTemplateString([NotNull] LGFileParser.MultilineTemplateStringContext context)
```

Comment:
I don't know why we need this? the whole point of rewriting multiple-line into tokens is to reuse the logic? right?